rake: release_task: Enclose the version in double quotes

abetomo opened the pull request:
If the version is handled as a number, the rake task will fail, so explicitly set it to a string.
    
For example, Mroonga version: 14.14.

abetomo committed:
rake: Fix version substitution

Double quotes are also supported.

Both of the following cases can be substituted:

* `mroonga_version: "14.12"`
* `groonga_version: 14.1.3`

abetomo commented:
```diff
  jekyll_config.gsub!(/^(#{escaped_product_id}_version: ).+$/) do
-     "#{$1}#{latest_version}"
+     "#{$1}\"#{latest_version}\""
  end
```

バージョンは `"` で囲むようにする↑くらいでも十分そう？

komainu8 commented:
> ```diff
>   jekyll_config.gsub!(/^(#{escaped_product_id}_version: ).+$/) do
> -     "#{$1}#{latest_version}"
> +     "#{$1}\"#{latest_version}\""
>   end
> ```
> 
> バージョンは `"` で囲むようにする↑くらいでも十分そう？

そうですね。↑で十分なように思います。

abetomo committed:
rake: release_task: Enclose the version in double quotes

If the version is handled as a number, the rake task will fail, so explicitly set it to a string.

For example, Mroonga version: 14.14.
